### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by Nicooow** [Tarh-76/Python-package-vacuum-map-parser-ijai#1]
Hi there,

Do you still have access to your Github account? I'd like to contact you

**Comment by Tarh-76** [comment]
Hi Nicow,
Looks like I do. Contact confirmed. 😉 How can I help?

**Comment by Nicooow** [comment]
Do you have discord ? if yes can you add me ?  @ nicow

**Comment by Tarh-76** [comment]
Unfortunately I don't, even worse: it is blocked here. Tg maybe? (Tarh007)

**Comment by PiotrMachowski** [comment]
Hi!

I will allow myself to also join this conversation :) 
Do you plan to release this package to pypi? I would like to include it in the next version of Map Extractor

I can also fork your repo and publish it by myself if you prefer that approach.

**Comment by Tarh-76** [comment]
Hi Piotr,
you're always more than welcome! Talking about releases of the ijai module I consider both approaches are fine. I would only like to get maksp86's PR merged as he had made quite an improvement to map postprocessing. Also, I'm not familiar with the method how to commit to pypi. I think I can figure it out though.

**Comment by PiotrMachowski** [comment]
This job publishes a release in PyPi when you create a release in GitHub. 
https://github.com/PiotrMachowski/Python-package-vacuum-map-parser-roborock/blob/master/.github/workflows/release.yml

**Comment by maksp86** [comment]
> I would like to include it in the next version of Map Extractor

Check commits in Tarh`s fork of Extractor base, because this map parser requires some additional data like MAC

_Also, if you don't mind and have some spare time, please consider adding a way to change the base font for room names :)
This would be more than useful for non-English speaking countries._

**Comment by PiotrMachowski** [comment]
> Check commits in Tarh`s fork of Extractor base, because this map parser requires some additional data like MAC

Yeah, I've noticed it. I have such data in the next version of Map Extractor (it has been completely rewritten; for a third time I think...) and it will be fully configurable from the UI.

> Also, if you don't mind and have some spare time, please consider adding a way to change the base font for room names :)
> This would be more than useful for non-English speaking countries.

Yeah, current font is basically useless. I know how to adjust it, I might be able to include it in the next release. At this moment I just disabled drawing room names by default (and I think I will leave it this way in the final version anyway).
